simpleflu/README.md

```diff
@@ -1,28 +1,23 @@
 # Simple Flu
 
-This example introduces the **Simple Flu** model, which is a standalone model of flu infection and the basis for the `../flu-with-behavior`, `../school-closure`, and `../vaccine` tutorials.
+This example introduces the **Simple Flu** model, a standalone model for influenza and the basis for the `../flu-with-behavior`, `../school-closure`, and `../vaccine` tutorials.
```

Comment:
I think we should use the term "influenza" instead of "flu", but I am okay with continuing to call this the "simple flu" model.